fair/tools/unifiedtools.py

```diff
@@ -196,6 +199,86 @@ def create_output_dataframes(inp_df, gas_np, RF_np, T_np, alpha_np, ext_forcing_
     )
     return gas_df, RF_df, T_df, alpha_df
 
+def create_output_dataframe_aimc_compliant(inp_df, gas_np, RF_np, T_np, alpha_np, ext_forcing_np):
+    """
+    TODO: docstring
+    """
+
+    units = Units()
+
+    variable_array = inp_df.timeseries().index.levels[3].to_numpy()
+    split_variable_array = np.array([variable_array[i].split('|',1) for i in range(len(variable_array))])
+
+    gas_array = split_variable_array[:,1]
+    function_array = split_variable_array[:,0]
+    unique_function_array = np.unique(function_array)
+    if len(unique_function_array) > 1:
+        raise Exception('Error: More than one type of input passed')
+
+    inp_function = unique_function_array[0]
+
+    output_function = "Atmospheric Concentrations" if str.lower(inp_function[0]) == 'e' else "Emissions"
+
+    model_region_scenario_array = np.unique(inp_df.timeseries().index.droplevel(('unit','variable')).to_numpy())
+    if len(model_region_scenario_array) > 1:
+        raise Exception('Error: More than one Model, Region + Scenario combination input passed')
+    model_region_scenario = np.array(model_region_scenario_array[0])
+
+    time_index = inp_df.timeseries().columns
+
+    data_array = np.array([[]])
+    for i in range(len(gas_array)):
+        gas_name = gas_array[i]
+        
+        gas_output_variable = output_function + "|" + gas_name
+        RF_output_variable = "Effective Radiative Forcing|" + gas_name
+        alpha_output_variable = "Alpha|" + gas_name
+
+        gas_output_unit = units[gas_output_variable]
+        RF_output_unit = units[RF_output_variable]
+        alpha_output_unit = units[alpha_output_variable]
+
+        gas_output_array = np.append(model_region_scenario,[gas_output_variable, gas_output_unit])
+        RF_output_array = np.append(model_region_scenario,[RF_output_variable, RF_output_unit])
+        alpha_output_array = np.append(model_region_scenario,[alpha_output_variable, alpha_output_unit])
+
+        gas_output_array = np.append(gas_output_array,gas_np[i])
+        RF_output_array = np.append(RF_output_array,RF_np[i])
+        alpha_output_array = np.append(alpha_output_array,alpha_np[i])
+
+        data_array = np.append(data_array,[gas_output_array],axis=int(i==0))
+        data_array = np.append(data_array,[RF_output_array],axis=0)
+        data_array = np.append(data_array,[alpha_output_array],axis=0)
+
+    ext_RF_output_variable = "Effective Radiative Forcing|External Forcing"
```

Comment:
```suggestion
    ext_RF_output_variable = "Effective Radiative Forcing|Other"
```

or "Prescribed"? External Forcing doesn't make any sense to me and will confuse people I suspect